r2r/src/nodes.rs

```diff
@@ -1651,6 +1757,22 @@ impl Timer {
             Err(Error::RCL_RET_TIMER_INVALID)
         }
     }
+
+    /// Transforms this timer stream to a [`Future`] calling the given `callback` on each tick.
+    ///
+    /// The callback execution is traced by r2r_tracing.
+    ///
+    /// This function should be called immediately after the timer is created.
```

Comment:
This function should be called before dropping the timer's node. Otherwise, the trace data might be inconsistent.